interface/src/mvd-hosting.d.ts

```diff
@@ -89,17 +89,20 @@ declare namespace MVDHosting {
     performLogin(username: string, password: string): Observable<Response>;
   }
 
-  export interface NotificationManagerInterface {
-    push(notification: Notification): void;
-    pop(): Notification | void;
-    getAll(): Notification[] | void;
-    getAllByCategory(type: MVDHosting.NotificationType): Notification[] | void;
+  export interface ZoweNotificationManagerInterface {
+    _setURL(url: string): void;
```

Comment:
interfaces should never have _ functions in them. You use these functions because you secretly know they exist, not that you want people to know they exist. In typescript, doing (notificationmanager as any)._setURL can make typescript OK with the fact that you didnt declare it formally.